src/components/ListingDetails/index.tsx

```diff
@@ -322,7 +317,15 @@ export default function ListingDetails({
           <P>{listingData.summary || 'Not Available'}</P>
         </BorderedSection>
       </InfoContainer>
-      <BorderedSection>{interestSection}</BorderedSection>
+      <BorderedSection>
+        <Flex $gap="40px">
+          <ProfileMatch
+            listingData={listingData}
+            interpretation={interpretation}
+          />
+          {interestSection}
```

Comment:
Looks like the `ProfileMatch` component is still visible when the user is not logged in or hasn't completed onboarding. Let's go ahead and add the `ProfileMatch` component in the `interestSection` so that it's also hidden in these cases.

**Not logged in:**
<img width="995" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/b7e4bdd3-b6d3-4132-baf6-71a54551aabc">

**Not onboarded**
<img width="985" alt="Profile Match" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/d7ce76b1-e174-4f32-b81a-fbf41859047e">

Reply:
good catch! fixed